components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -20,15 +20,16 @@
 static CGFloat const kChipsSpacing = 0.0f;
 static CGFloat const kTextToEnterPlaceholderLength = 16.0f;
 
-@interface MDCChipTextField () <MDCChipTextFieldScrollViewDataSource>
+@interface MDCChipTextField () <MDCChipTextFieldScrollViewDataSource,
+                                MDCChipTextFieldScrollViewDelegate>
 
 @property(nonatomic, strong) MDCChipTextFieldScrollView *chipsContainerView;
 
 @property(nonatomic, readwrite, weak) NSLayoutConstraint *chipContainerViewConstraintLeading;
 @property(nonatomic, readwrite, weak) NSLayoutConstraint *chipContainerViewConstraintTrailing;
 @property(nonatomic) CGFloat chipContainerViewConstraintTrailingConstant;
 // Chip view models
-@property(nonatomic, strong) NSMutableArray<MDCChipView *> *chipViews;
+@property(nonatomic, readwrite, copy) NSMutableArray<MDCChipView *> *mutableChipViews;
```

Comment:
Why are we using a different name? Is it possible to use a mutable chipViews array in the implementation, which uses the same name as the readonly interface?

Reply:
I don't think it is possible to change the type for a property in implementation file.

Reply:
One way to avoid using 2 different names to define only a getter for chipViews - by making it an accessor method:
- NSArray<MDCChipView *> *chipViews;

Reply:
If you declare a public `readonly` NSArray, you can just return a copy of the internally-declared NSMutableArray in the custom getter.  Any reason not to do that?

Reply:
Eh. I am confused. I guess there were two issues you were talking about. 
@romoore Yes, and that is how it is currently implemented.
@galiak11 Directly exposing the accessor method of a private property is dangerous, and I think there will also be compiler issues if [self.chipsViews addObject:] is called in this case. Avoiding using two names is doable if we swap the synthesized instance with a mutable one, but I think using property is always a better practice in this case.

Reply:
@wenyuzhang666 So if there's no reason not to go with the current approach, why are you proposing changing it?

Reply:
Sorry. The changes in the PR was what I meant by `currently implemented`.

Reply:
Thanks for clearing that up. Makes sense. 👍